ckan/public/base/javascript/modules/autocomplete.js

```diff
@@ -50,7 +53,8 @@ this.ckan.module('autocomplete', function (jQuery) {
         formatNoMatches: this.formatNoMatches,
         formatInputTooShort: this.formatInputTooShort,
         dropdownCssClass: this.options.dropdownClass,
-        containerCssClass: this.options.containerClass
+        containerCssClass: this.options.containerClass,
+        tokenSeparators: eval(this.options.tokensep)
```

Comment:
Could you please parse this options manually? Inside constructor, for example? I understand, that eval doesn't do any  harm here, but anyway no one like `eval` :)

Reply:
I see. I took the opportunity to change the syntax of the parametrization (value of `data-module-tagsep`) to something simpler. Now only single characters are possible as separation markers, but that is OK, I believe.